dashboard/config/locales/en.yml

```diff
@@ -1155,6 +1155,7 @@ en:
       copy_code: 'Please copy and paste this code into the Maker App to login to Code.org.'
       error_token_expired: 'Token has expired. Please log in again for a new token.'
       error_wrong_provider: 'Wrong provider. Please make sure you are logging in with Google.'
+      error_no_linked_google: 'Wrong provider. Please make sure you are logging in with Google or you have linked your Google account to your Code.org account'
```

Comment:
[commenting on the fact that this change is present in 2 different strings files] did you have to update both files manually? or was one automatically updated as part of a build/commit/etc hook?

Reply:
oops, one of these is not necessary, I'll remove it